module/apmhttp/handler_test.go

```diff
@@ -502,6 +502,50 @@ func TestHandlerTraceparentHeader(t *testing.T) {
 	}
 }
 
+func TestHandleContinuationStrategy(t *testing.T) {
```

Comment:
I would prefer that we have this test in `package apm`, since that's where the logic lives. Can we test the functionality using `Tracer.StartTransactionOptions`? Maybe as a table test, with a combination of trace states with/without es, and the continuation strategy.

Reply:
I've reworked the test :+1:

Reply:
I've moved the test and expanded the test cases :+1: